css/src/components/message.scss

```diff
@@ -0,0 +1,50 @@
+.message {
+	display: grid;
+	grid-template:
+		'. message-time'
+		'message-persona message-content';
+	grid-template-columns: max-content 1fr;
+	gap: $layout-1;
```

Comment:
All of these values (gap, font-size, margin, padding, etc) should be assigned to variables for flexibility. Check out any component (here is the [Persona](https://github.com/microsoft/atlas-design/blob/main/css/src/components/persona.scss) as an example) for reference.

Reply:
updated